src/test/resources/smoke/method.vpr

```diff
@@ -0,0 +1,5 @@
+method test()
```

Comment:
A test with an abstract method would be good

Reply:
-> d19e4e7f